From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 24 Nov 2012 11:39:58 +0100
Subject: Strip version number from application before installation

Strip thunderbird's version number from the installation directories so
we have a fixed location for extensions that build against it.
---
 config/baseconfig.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/baseconfig.mk b/config/baseconfig.mk
index 8f3e920..47c442a 100644
--- a/config/baseconfig.mk
+++ b/config/baseconfig.mk
@@ -3,6 +3,9 @@
 # We thus use INCLUDED_AUTOCONF_MK to enable/disable some parts depending
 # whether a normal build is happening or whether the check is running.
 installdir = $(libdir)/$(MOZ_APP_NAME)
+includedir := $(includedir)/$(MOZ_APP_NAME)
+idldir = $(datadir)/idl/$(MOZ_APP_NAME)
+sdkdir = $(libdir)/$(MOZ_APP_NAME)-devel
 ifeq (.,$(DEPTH))
 DIST = dist
 else
